src/app/components/MessageBanner/index.tsx

```diff
@@ -74,6 +74,7 @@ const Banner = forwardRef(
                   alt=""
                   src={image.replace('{width}', 'raw')}
                   placeholder={false}
+                  aspectRatio={[16, 9]}
```

Comment:
Do any of the other components which use an Image directly require the aspectRatio/dimensions to be set?

Reply:
I checked and this was the only one. 

There were some Frosted Promo tests that fell back on the default aspect ratio but they either had no image or only received partial props. 

As far as I can tell that Component should recieve an Image with a width and a height provided.